components/automate-chef-io/content/docs/backup.md

```diff
@@ -192,7 +192,11 @@ Restore Chef Automate to an instance on which Automate is not already installed.
 
 #### Prerequisites
 
-1. Install the `chef-automate` command line utility on the restore host.
+1. On the restore host, download and unzip the Chef Automate command-line utility:
```

Comment:
If we'd rather link to the install docs instead of writing this out, that's fine.